.github/workflows/scripts/code_coverage_disable_check.py

```
@@ -85,7 +83,8 @@
                         file_path = os.path.join(root, file_name)
                         if has_code_coverage_disable(file_path):
                             print(
-                                f"""File {file_path} contains code coverage disable statement."""
+                                f"""\
+File {file_path} contains code coverage disable statement."""
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)